### PR TITLE
placeholders added for custom fields refs #8197

### PR DESCRIPTION
**Pull request by sehrnadeem1**
http://pm.7vals.com/issues/8197

Rails PR: https://github.com/7Vals/ezofficeinventory/pull/7871
Ember PR: https://github.com/7Vals/ember_webview_app/pull/258

Placeholders added for custom fields.

### CODE REVIEW

**Comment by sehrnadeem1** [comment]
Cycle 0

File: app/views/custom_attributes/custom_attribute_definitions/_form_fields_only.html.erb [hunk 1]
```diff
@@ -16,6 +16,14 @@
   <%= f.text_field :default_value %>
 </div>
 
+<div class="custom-attribute-definition title <%=titleClass%>">Placeholder:
```
Comment (mairamuneer39) [inline app/views/custom_attributes/custom_attribute_definitions/_form_fields_only.html.erb]: space
line 22 as well

File: app/views/custom_attributes/custom_attribute_definitions/_form_fields_only.html.erb [hunk 2]
```diff
@@ -16,6 +16,14 @@
   <%= f.text_field :default_value %>
 </div>
 
+<div class="custom-attribute-definition title <%= titleClass %>">Placeholder:
```
Comment (saqibbukhari) [inline app/views/custom_attributes/custom_attribute_definitions/_form_fields_only.html.erb]: add migration 
add version in app code

Comment (sehrnadeem1) [reply app/views/custom_attributes/custom_attribute_definitions/_form_fields_only.html.erb]: done

**Pull request merged**